recipes/sdsl-lite/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sdsl-lite" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simongog/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz.offline.install.gz
+  fn: {{ name }}.tar.gz # necessary because of the nonstandard file extension from upstream
+  sha256: 2f192977b47406ae8992292c7e841ef23d4656bf72f6140540bed53af68e06ed
+
+build:
+  number: 0
+  script: "$SRC_DIR/install.sh $PREFIX"  # [unix]
```

Comment:
Instead of doing this.
Please, create a file called ``build.sh`` and ``build.bat`` and ``conda`` will already call this file in your recipe 
Inside of each of this file you put the commands to be executed to build your package

Reply:
I thought the one-liners were preferred if possible? Upstream package already supplies these `install.sh` and `install.bat` files. The `build.sh' would just look like this:
```
#!/bin/sh
$SRC_DIR/install.sh $PREFIX
```
Isn't that better in the `meta.yaml`? Sorry for the questions, I'm used to the bioconda rules and this my first PR to conda-forge